Alerting: Customise OK notification priorities for Pushover notifier (#30169)
grafana/grafana: docs/sources/alerting/notifications.md

This behavior will become the default in a future version of Grafana.

> Using `dedup_key` tag will override Grafana generated `dedup_key` with a custom key.

### Pushover

> To set up Pushover, all you have to do is to provide a User key and an API token. You can follow [What is Pushover and how do I use it](https://support.pushover.net/i7-what-is-pushover-and-how-do-i-use-it) on how to generate them.

Reviewer: ```suggestion
To set up Pushover, you must provide a user key and an API token. Refer to [What is Pushover and how do I use it](https://support.pushover.net/i7-what-is-pushover-and-how-do-i-use-it) for instructions on how to generate them.
```

Setting | Description
---------- | -----------
API Token | Application token

> User key(s) | A comma-separated list of User keys

Reviewer: ```suggestion
User key(s) | A comma-separated list of user keys
```

Device(s) | A comma-separated list of devices
Priority | The priority alerting nottifications are sent

> OK priority | The priority OK nottifications are sent; if not set OK notifications will be sent with the priotiry set for alerting notifications 

Reviewer: ```suggestion
OK priority | The priority OK notifications are sent; if not set, then OK notifications are sent with the priority set for alerting notifications 
```

> Retry | How often (in seconds) the Pushover servers will send the same notification to the user. (minimun 30 seconds)

Reviewer: ```suggestion
Retry | How often (in seconds) the Pushover servers send the same notification to the user. (minimum 30 seconds)
```

Expire | How many seconds your notification will continue to be retried for (maximum 86400 seconds)
Alerting sound | The sound for alerting notifications

> OK sound | The sound for OK notifications

Reviewer: ```suggestion
OK sound | The sound for OK notifications

```
